frontend/src/LogInSignUp/LogInSignUp.jsx

```diff
@@ -33,7 +35,9 @@ const LogInSignUp = () =>{
                     <input type="password" placeholder="Password" />
                 </div>
 
-                {action==="Sign Up"?<div></div>:<div className="ForgotPassword">Forgot Password <span>Click Here!</span></div>}
+                {action==="Sign Up"?
+                <div></div>:
```

Comment:
are you planning to add smth else into this `<div>` element?

Reply:
No

Reply:
so in this case you can use `&&` operator which would render en element if the condition on the left side evaluates to true, so you can do smth like: `{action !== "Sign Up" && <div className="ForgotPassword">Forgot Password <span>Click Here!</span></div>}` so you are not rendering an empty `div`
Here is more info on `&&` operator: https://react.dev/learn/conditional-rendering (under `Logical AND operator (&&) `)